requirements/test.in

```diff
@@ -0,0 +1,13 @@
+# Requirements for test runs
+-c constraints.txt
+
+-r base.txt               # Core dependencies for the cookiecutter
+
+django-nose==1.4.6
+ddt
+coveralls
+mock
+pep8
+django-pyfs==2.0
+edx_lint==1.4.0
```

Comment:
Same here, since you are using constraint file, you should move the dependencies there.

Reply:
I just removed them and tests are working fine.